### ISSUE TITLE
`perform_inline` requires ActiveSupport `constantize` String extension

### ISSUE THREAD

**Issue by rgalanakis** [mperham/sidekiq#5510]
Ruby version: 2.7 to 3.1
Rails version: Not using
Sidekiq / Pro / Enterprise version(s): 6.4

I have some Sidekiq extensions extracted out of some consulting projects, and we use `perform_inline` to test some of them. I just ran into this:

https://github.com/mperham/sidekiq/blob/7656c350ce923f7aa9e3d25db4bd5e457cf78964/lib/sidekiq/worker.rb#L221

Sidekiq has its own `constantize` function but it looks like this assumes the ActiveSupport string extension method is available.

**Comment by mperham** [comment]
Ah right. It's fixed in 7.0 and if you are on Ruby 2.7+, you can monkeypatch like this:

```ruby
class String
  def constantize
    Object.const_get(self)
  end
end
```

**Comment by rgalanakis** [comment]
Thanks. This was easy to monkeypatch thankfully, so sounds like this can wait for the big upgrade. Thanks for the prompt response!
